test/test_utils.py

```diff
@@ -1,54 +1,110 @@
 import pytest
```

Comment:
Could you please add 1-1 simple tests for RSU, ESPP, and the rest of the new sheets? Just to verify that it works on basic examples.

Reply:
I'll add more tests for things with partially empty sheets. A full example for the new sheet format is already given.